app/src/main/java/com/github/se/project/model/profile/ProfilesRepositoryFirestore.kt

```diff
@@ -40,17 +40,22 @@ class ProfilesRepositoryFirestore(private val db: FirebaseFirestore) : ProfilesR
   }
```

Comment:
I know it was not directly you but I just figured out that we never remove the listener on the auth in init. It may cause memory leaks and crashs.